speculos/mcu/apdu.py

```diff
@@ -65,12 +76,12 @@ def recv_packet(self):
     def can_read(self, s, screen):
```

Comment:
Explicit `s` + typing

Reply:
Hu, maybe you catched a bug here. `screen` should be a `DisplayNotifier`, but `DisplayNotifier` does not have a `forward_to_app` method, that's a `Display` method. Good catch!